bundles/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/action/FroniusSymoInverterActions.java

```diff
@@ -166,4 +185,18 @@ public boolean addForcedBatteryChargingSchedule(ZonedDateTime from, ZonedDateTim
             QuantityType<Power> power) {
         return addForcedBatteryChargingSchedule(from.toLocalTime(), until.toLocalTime(), power);
     }
+
+    @RuleAction(label = "@text/actions.backup-reserved-battery-capacity.label", description = "@text/actions.backup-reserved-battery-capacity.description")
+    public @ActionOutput(type = "boolean", label = "Success") boolean setBackupReservedBatteryCapacity(
+            @ActionInput(name = "percent", label = "@text/actions.soc.label", description = "@text/actions.soc.description", required = true) int percent) {
+        FroniusSymoInverterHandler handler = this.handler;
+        if (handler != null) {
+            return handler.setBackupReservedBatteryCapacity(new PercentType(percent));
+        }
+        return false;
+    }
+
+    public boolean setBackupReservedBatteryCapacity(PercentType percent) {
+        return setBackupReservedBatteryCapacity(percent.intValue());
```

Comment:
Just to make my other comment more clear: Here you take `PercentType`, grab `intValue()` and pass that to the other overload, which then converts it back to a `PercentType`, provides it to the API class which in turn again takes the `intValue()`.